### ISSUE TITLE
Erroneous violation entry in function-url-scheme-whitelist documentation

### ISSUE THREAD

**Issue by BenjaminWFox** [stylelint/stylelint#3115]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Documentation fix

> Which rule, if any, is this issue related to?

function-url-scheme-whitelist

> What CSS is needed to reproduce this issue?

N/A

> What stylelint configuration is needed to reproduce this issue?

N/A

> Which version of stylelint are you using?

8.4.0

> How are you running stylelint: CLI, PostCSS plugin, Node API?

N/A

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

Correct documentation showing correct violation & non-violation examples

> What actually happened (e.g. what warnings or errors you are getting)?

A duplicate entry existed in both the violations and non-violations sections

```md
README.md

Given:

["data", "/^http/"]

The following patterns are considered violations:

a { background-image: url('http://www.example.com/file.jpg'); }
```

**Comment by ntwb** [comment]
Fixed in #3116
